config/index.conf.php

```diff
@@ -21,12 +21,21 @@
 declare(strict_types=1);
 
 $dynamicTemplates = [
+    [
+        "Calendar" => [
+            "match" => "Calendar_*",
+            "match_mapping_type" => "string",
+            "mapping" => [
+                "type" => "keyword"
+            ]
+        ]
+    ],
     [
         "HTMLArea" => [
             "match" => "HTMLArea_*",
             "match_mapping_type" => "string",
             "mapping" => [
-                "type" => "text"
+                "type" => "keyword"
```

Comment:
https://github.com/oat-sa/extension-tao-advanced-search/pull/68/files#r971643516: I'm not sure if we should keep these as "text" to be able to search inside their contents.